### ISSUE TITLE
train

### ISSUE THREAD

**Issue by mp0312** [xiaoaoran/FPS-Net#7]
When I run train.sh, it gets stuck straight into the main and doesn't move at all, why is that? Looking forward to your reply!
------------------------------------------Here's what the console shows-------------------------------

No pretrained directory found.
Copying files to ./logs/2024-5-19-09:20/ for further reference.
Sequences folder exists! Using sequences from /home/hgy01/3742/sequences
parsing seq 00
parsing seq 01
parsing seq 02
parsing seq 03
parsing seq 04
parsing seq 05
parsing seq 06
parsing seq 07
parsing seq 09
parsing seq 10
Using 19130 scans from sequences [0, 1, 2, 3, 4, 5, 6, 7, 9, 10]
Sequences folder exists! Using sequences from /home/hgy01/3742/sequences
parsing seq 08
Using 4071 scans from sequences [8]
Loss weights from content:  tensor([  0.0000,  22.9317, 857.5627, 715.1100, 315.9618, 356.2452, 747.6170,
        887.2239, 963.8915,   5.0051,  63.6247,   6.9002, 203.8796,   7.4802,
         13.6315,   3.7339, 142.1462,  12.6355, 259.3699, 618.9667])
Using ms_rdb_unet backbone
Using RCNN decoder
Total number of parameters:  55726132
Total number of parameters requires_grad:  55726132
Param encoder  48227232
Param decoder  7493120
Param head  5780
No path to pretrained, using random init.
Training in device: GPU  [0, 1]
Let's use 2 GPUs!
Ignoring class  0  in IoU evaluation
[IOU EVAL] IGNORE:  tensor([0])
[IOU EVAL] INCLUDE:  tensor([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16, 17, 18,
        19])
~
